elm.json

```diff
@@ -1,310 +1,16 @@
-    "ZondIcons.MusicNotes",
-    "ZondIcons.MusicPlaylist",
-    "ZondIcons.NavigationMore",
-    "ZondIcons.Network",
-    "ZondIcons.NewsPaper",
-    "ZondIcons.Notification",
-    "ZondIcons.Notifications",
-    "ZondIcons.NotificationsOutline",
-    "ZondIcons.Paste",
-    "ZondIcons.Pause",
-    "ZondIcons.PauseOutline",
-    "ZondIcons.PauseSolid",
-    "ZondIcons.PenTool",
-    "ZondIcons.Phone",
-    "ZondIcons.Photo",
-    "ZondIcons.PhpElephant",
-    "ZondIcons.Pin",
-    "ZondIcons.Play",
-    "ZondIcons.PlayOutline",
-    "ZondIcons.Playlist",
-    "ZondIcons.Plugin",
-    "ZondIcons.Portfolio",
-    "ZondIcons.Printer",
-    "ZondIcons.Pylon",
-    "ZondIcons.Question",
-    "ZondIcons.Queue",
-    "ZondIcons.Radar",
-    "ZondIcons.RadarCopy_2",
-    "ZondIcons.Radio",
-    "ZondIcons.Refresh",
-    "ZondIcons.Reload",
-    "ZondIcons.Reply",
-    "ZondIcons.ReplyAll",
-    "ZondIcons.Repost",
-    "ZondIcons.SaveDisk",
-    "ZondIcons.ScreenFull",
-    "ZondIcons.Search",
-    "ZondIcons.Send",
-    "ZondIcons.Servers",
-    "ZondIcons.Share",
-    "ZondIcons.ShareAlt",
-    "ZondIcons.Share_01",
-    "ZondIcons.Shield",
-    "ZondIcons.ShoppingCart",
-    "ZondIcons.ShowSidebar",
-    "ZondIcons.Shuffle",
-    "ZondIcons.StandBy",
-    "ZondIcons.StarFull",
-    "ZondIcons.Station",
-    "ZondIcons.StepBackward",
-    "ZondIcons.StepForward",
-    "ZondIcons.Stethoscope",
-    "ZondIcons.StoreFront",
-    "ZondIcons.StrokeWidth",
-    "ZondIcons.SubdirectoryLeft",
-    "ZondIcons.SubdirectoryRight",
-    "ZondIcons.Swap",
-    "ZondIcons.Tablet",
-    "ZondIcons.Tag",
-    "ZondIcons.Target",
-    "ZondIcons.TextBox",
-    "ZondIcons.TextDecoration",
-    "ZondIcons.Thermometer",
-    "ZondIcons.ThumbsDown",
-    "ZondIcons.ThumbsUp",
-    "ZondIcons.Ticket",
-    "ZondIcons.Time",
-    "ZondIcons.Timer",
-    "ZondIcons.ToolsCopy",
-    "ZondIcons.Translate",
-    "ZondIcons.Trash",
-    "ZondIcons.Travel",
-    "ZondIcons.TravelBus",
-    "ZondIcons.TravelCar",
-    "ZondIcons.TravelCase",
-    "ZondIcons.TravelTaxiCab",
-    "ZondIcons.TravelTrain",
-    "ZondIcons.TravelWalk",
-    "ZondIcons.Trophy",
-    "ZondIcons.Tuning",
-    "ZondIcons.Upload",
-    "ZondIcons.Usb",
-    "ZondIcons.User",
-    "ZondIcons.UserAdd",
-    "ZondIcons.UserGroup",
-    "ZondIcons.UserSolidCircle",
-    "ZondIcons.UserSolidSquare",
-    "ZondIcons.Vector",
-    "ZondIcons.VideoCamera",
-    "ZondIcons.ViewCarousel",
-    "ZondIcons.ViewColumn",
-    "ZondIcons.ViewHide",
-    "ZondIcons.ViewList",
-    "ZondIcons.ViewShow",
-    "ZondIcons.ViewTile",
-    "ZondIcons.VolumeDown",
-    "ZondIcons.VolumeMute",
-    "ZondIcons.VolumeOff",
-    "ZondIcons.VolumeUp",
-    "ZondIcons.Wallet",
-    "ZondIcons.Watch",
-    "ZondIcons.Window",
-    "ZondIcons.WindowNew",
-    "ZondIcons.WindowOpen",
-    "ZondIcons.Wrench",
-    "ZondIcons.YinYang",
-    "ZondIcons.ZoomIn",
-    "ZondIcons.ZoomOut"
-  ],
+  "exposed-modules": ["ZondIcons"],
```

Comment:
Maybe one thing to reconsider again is whether to go with `Zondicons` instead of  `ZondIcons` since that's also how it's written on the website.